WordPress/src/main/res/layout/restore_details_fragment.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
```

Comment:
1) Minor (🔍): This layout file can be reformatted.
2) Question (❓): On the `BackupDownload` equivalent layout file, there is an unused `android:id` and an extra `android:padding`. I am guessing we can remove the unused `android:id` from there to increase consistency between the two layout files. However, I am not sure about the `android:padding` and whether this is required for the `Restore` case.

Reply:
Great comment - I'll be focusing on styling next week. I am aware of the padding already. :)